step.yml

```diff
@@ -103,21 +116,30 @@ inputs:
 outputs:
   - APPCENTER_DEPLOY_STATUS:
     opts:
-      title: "Deployment status"
+      title: Deployment status
       summary: "Deployment status: 'success' or 'failed'"
       description: "Deployment status: 'success' or 'failed'"
   - APPCENTER_DEPLOY_INSTALL_URL:
     opts:
-      title: Install URL of the newly deployed version
-      summary: ""
-      description: ""
+      title: Install page URL
+      summary: Install page URL of the newly deployed version
+      description: |-
+        Install page URL of the newly deployed version.
+
+        App Center sign in is required.
   - APPCENTER_DEPLOY_DOOWNLOAD_URL:
     opts:
-      title: Download URL of the newly deployed version
+      title: Download URL
       summary: Download URL of the newly deployed version
-      description: Download URL of the newly deployed version
+      description: |-
+        Download URL of the newly deployed version.
+        
+        App Center sign in is required.
```

Comment:
no 🙂 it is usually an itms:// link that installs on the iOS devices instantly. I can't see how it would integrate if the url requires login. 🤔